src/ansys_sphinx_theme/static/404.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+Oops!
+=====
+This is unexpected. The page you are requesting does not exist.
+
+If this page should exist, please contact `{{ theme_contact_mail }} <mailto:{{ theme_contact_mail }}`_.
```

Comment:
Considering that some projects may not be linked to GitHub (see #38), we agreed on adding a new theme variable named `contact_mail` which is used in the `404.rst` file.

Reply:
#OpeningPRsEverywhere